test/jdk/java/awt/PrintJob/PrintCheckboxTest/PrintCheckboxManualTest.java

```diff
@@ -22,84 +22,50 @@
  */
 
 /*
-  @test
-  @bug 5045936 5055171
-  @summary Tests that there is no ClassCastException thrown in printing
-   checkbox and scrollbar with XAWT
-  @key printer
-  @run applet/manual=yesno PrintCheckboxManualTest.html
+ * @test
+ * @bug 5045936 5055171
+ * @summary Tests that there is no ClassCastException thrown in printing
+ *          checkbox and scrollbar with XAWT
```

Comment:
In case it is decided to extend this test to all platforms then XAWT can be removed from test summary.

```suggestion
 *          checkbox and scrollbar
```